.github/ISSUE_TEMPLATE/WG_member_request.yaml

```diff
@@ -0,0 +1,50 @@
+name: Working Group Member Request
+description: Request membership to serve on a PowerShell Working Group
+title: Working Group Member Request
+labels: [WG-NeedsReview, Needs-Triage]
+body:
+- type : dropdown
+  id : request_type
+  validations:
+    required: true
+  attributes:
+    label: Name of Working Group you are requesting to join?
+    description: >-
+      Please select the name of the working group you are requesting to join. (Select one)
+    options:
+      - "Engine"
+      - "Remoting"
+      - "Cmdlets and Modules"
+      - "Security"
```

Comment:
I think we're keeping Security as Msft only for now

Reply:
Cool, I'll remove security